quiz.js

```diff
@@ -7,26 +7,22 @@ let quizTime = 0;
 let numWrong = 0;
 let numCorrect = 0;
 
-for (i = 0; i < 3; i++) {
+for (let i = 0; i < questions.length; i++) {
     let difficulty = 1;
-    tstart = new Date().getTime();
     //console.log(tstart);
-    let userAnswer = prompt(questions[i].question);
-    if (typeof(userAnswer) === "string") {
```

Comment:
`prompt` will always return a string (even if the user types in `42` it will return the string `"42"`, so this check is not necessary.